### DISCUSSION TITLE
Fred's ImageMagick scripts down forever, or new home ?

### DISCUSSION THREAD

**Discussion by myicq** [ImageMagick/ImageMagick#6548] [Help]
### ImageMagick version

7.x

### Operating system, version and so on

Any

### Description

Recently, the excellent repository of "Fred's ImageMagick Scripts" [http://www.fmwconcepts.com/](fmwconcepts.com)  returns "Account Suspended". 

Is it coming back, has the collection found a new home, or is it gone forever ? 

Curious what happened. 

If any help is needed to restore the collection, please reach out.

**Comment by fmw42** [comment]
It was suspended without notice for too much disk usage. I contacted Godaddy and after a week, we updated my account to a larger disk storage. It has been back for several weeks now.

**Reply by fmw42** [reply]
Looks like Godaddy has suspended it again. I will have to contact them. There is no reason for it now.

**Reply by urban-warrior** [reply]
Fred, we can host your scripts @ imagemagick.org if you prefer.
